coordinator/src/main/scala/com/socrata/datacoordinator/resources/BasicSodaResource.scala

```diff
@@ -11,7 +11,7 @@ import org.slf4j.Logger
 
 abstract class BasicSodaResource extends SodaResource {
 
-  protected val log: Logger
+  protected val log: Logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
```

Comment:
Didn't we intentionally not instantiate logger in abstract classes so that log lines are more tightly associated with derived classes?

Reply:
I think this should do the right thing? i.e. get the class of the class inheriting. I can double check

Reply:
(verified that is works correctly)